ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Decision/Add.cshtml.cs

```diff
@@ -27,90 +29,78 @@ public class AddPageModel : AbstractPageModel
 		private readonly ILogger<AddPageModel> _logger;
 
 		[BindProperty]
-		public CreateDecisionRequest CreateDecisionDto { get; set; }
+		public CreateDecisionRequest Decision { get; set; }
 
-		[BindProperty]
-		public bool DecisionTypeNoticeToImprove { get; set; }
-
-		[BindProperty]
-		public bool DecisionTypeSection128 { get; set; }
-
-		[BindProperty]
-		public bool DecisionTypeQualifiedFloatingCharge { get; set; }
-
-		[BindProperty]
-		public bool DecisionTypeNonRepayableFinancialSupport { get; set; }
-
-		[BindProperty]
-		public bool DecisionTypeRepayableFinancialSupport { get; set; }
-
-		[BindProperty]
-		public bool DecisionTypeShortTermCashAdvance { get; set; }
-
-		[BindProperty]
-		public bool DecisionTypeWriteOffRecoverableFunding { get; set; }
-
-		[BindProperty]
-		public bool DecisionTypeOtherFinancialSupport { get; set; }
-
-		[BindProperty]
-		public bool DecisionTypeEstimatesFundingOrPupilNumberAdjustment { get; set; }
+		public int NotesMaxLength => 2000;

```

Comment:
Just to make things a bit easier. Do we need the max note length here ?
If we do, can we make the constants on the DTO public and reference them? That we can avoid changing this page if the length gets increased

Reply:
Will check and will add if required

Reply:
added constant